docs/5.community/6.roadmap.md

```diff
@@ -58,3 +58,32 @@ Auth           | Planned             | 3.x          | `nuxt/auth` to be announce
 Image          | Active              | 2.x and 3.x  | [nuxt/image](https://github.com/nuxt/image) | Nuxt 3 support is in progress: [nuxt/image#548](https://github.com/nuxt/image/discussions/548)
 Telemetry      | Active              | 2.x and 3.x  | [nuxt/telemetry](https://github.com/nuxt/telemetry/) | Nuxt 3 is supported. Stats to be public soon!
 I18n           | Active              | 2.x and 3.x  | [nuxt-modules/i18n](https://github.com/nuxt-modules/i18n) | See [nuxt-modules/i18n#1287](https://github.com/nuxt-modules/i18n/discussions/1287) for Nuxt 3 support
+
+## 📅 Long time support (LTS) versions
+
+### Current LTS version
+
+The current LTS is [V3.5](https://github.com/nuxt/nuxt/releases/tag/v3.5.3)
+
+### Next LTS version
+
+The next LTS is [V3.6](https://github.com/nuxt/nuxt/releases/tag/v3.6.1)
+
+Version number                                           | Status      | Initial release | Active LTS Start | Maintenance LTS Start | End Of Life  | Vue version | Minimal node version
+---------------------------------------------------------|-------------|-----------------|------------------|-----------------------|--------------|-------------|---------------------
+[V3.2](https://github.com/nuxt/nuxt/releases/tag/v3.2.3) | Unsupported | 02-09-2022      | 02-28-2022       | 03-14-2022            | 04-02-2022   | V3.2        | V16
+[V3.3](https://github.com/nuxt/nuxt/releases/tag/v3.3.3) | Maintenance | 03-14-2022      | 04-02-2022       | 04-11-2022            | 04-28-2022   | V3.2        | V16
+[V3.4](https://github.com/nuxt/nuxt/releases/tag/v3.4.3) | Maintenance | 04-11-2022      | 04-28-2022       | 05-16-2022            | 06-06-2022   | V3.2        | V16
+[V3.5](https://github.com/nuxt/nuxt/releases/tag/v3.5.3) | Current     | 05-16-2022      | 06-06-2022       | 06-23-2022            | 07-14-2022   | V3.3        | V16
+[V3.6](https://github.com/nuxt/nuxt/releases/tag/v3.6.1) | Active      | 06-23-2022      | 07-14-2022       | 07-28-2022            | 08-09-2022   | V3.3        | V16
+V3.7                                                     | Scheduled   | 07-28-2022      | 08-07-2022       | 08-21-2022            | 09-04-2022   | ~           | V18
```

Comment:
I noticed that there has been no update here for two weeks now, and I'm submitting a suggestion change to be able to close this pr.

```suggestion
Version number                                           | Status      | Initial release | Active LTS Start | Maintenance LTS Start | End Of Life  | Vue version | Minimal node version
---------------------------------------------------------|-------------|-----------------|------------------|-----------------------|--------------|-------------|---------------------
[V3.2](https://github.com/nuxt/nuxt/releases/tag/v3.2.3) | Unsupported | 2022-02-09      | 2022-02-28       | 2022-03-14            | 2022-04-02   | V3.2        | V16
[V3.3](https://github.com/nuxt/nuxt/releases/tag/v3.3.3) | Maintenance | 2022-03-14      | 2022-04-02       | 2022-04-11            | 2022-04-28   | V3.2        | V16
[V3.4](https://github.com/nuxt/nuxt/releases/tag/v3.4.3) | Maintenance | 2022-04-11      | 2022-04-28       | 2022-05-16            | 2022-06-06   | V3.2        | V16
[V3.5](https://github.com/nuxt/nuxt/releases/tag/v3.5.3) | Current     | 2022-05-16      | 2022-06-06       | 2022-06-23            | 2022-07-14   | V3.3        | V16
[V3.6](https://github.com/nuxt/nuxt/releases/tag/v3.6.1) | Active      | 2022-06-23      | 2022-07-14       | 2022-07-28            | 2022-08-09   | V3.3        | V16
V3.7                                                     | Scheduled   | 2022-07-28      | 2022-08-07       | 2022-08-21            | 2022-09-04   | ~           | V18
```

Reply:
But I find it very easy to document out of date LTS versions here... Maybe there needs to be an automated tool to generate the tables?